src/calculateRentalCost.js

```diff
@@ -3,8 +3,23 @@
  *
  * @return {number}
  */
+
+const basePrice = 40;
+const threeDays = 3;
+const threeDaysDiscount = 20;
+const sevenDays = 7;
+const sevenDaysDiscount = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < threeDays) {
+    return basePrice * days;
+  }
+
+  if (days < sevenDays) {
+    return basePrice * days - threeDaysDiscount;
+  }
+
+  return basePrice * days - sevenDaysDiscount;
```

Comment:
This return statement correctly applies the `sevenDaysDiscount` for rentals of 7 or more days. However, to maintain consistency and readability, consider using a variable for the total cost as suggested earlier.